lib/git.zsh

```diff
@@ -19,6 +19,7 @@ function git_prompt_info() {
 
   local ref
   ref=$(__git_prompt_git symbolic-ref --short HEAD 2> /dev/null) \
+  || ref=$(__git_prompt_git describe --exact-match HEAD 2> /dev/null) \
```

Comment:
Should we use `command git describe --tags --exact-match HEAD`? According to git documentation, this would also match unannotated tags, I'd expect this kind of tags to match also. What do you think?
```
       --tags
           Instead of using only the annotated tags, use any tag found in refs/tags
           namespace. This option enables matching a lightweight (non-annotated) tag.
```

Reply:
For example, tags created by Github when doing a release are lightweight tags.
I found more information about this topic (that was totally unknown) here: https://stackoverflow.com/questions/4971746/why-should-i-care-about-lightweight-vs-annotated-tags

Reply:
Agreed, I'll make the changes.

Reply:
```suggestion
  || ref=$(__git_prompt_git describe --tags --exact-match HEAD 2> /dev/null) \
```